src/main/java/exceptions/NotFoundInArrayList.java

```diff
@@ -0,0 +1,7 @@
+package exceptions;
+
+public class NotFoundInArrayList extends  Exception{
```

Comment:
Also this is not a good naming convention for an exception. Custom exceptions should be postfixed with Exception and should be named appropriately with focus on your functional requirements and not java specific error.
F.e. ProductNotFoundException.